src/repository/base.repository.js

```diff
@@ -64,6 +64,17 @@ class BaseRepository {
   }
 
   /**
+   * @static
+   * @param {object} data - object of containing the relevant data
+   * @param {object} options - column options
+   * @returns {object} - returns an database object
+   * @memberof BaseRepository
+   */
+  static async updateField(data, options) {
```

Comment:
Please check the current baseRepository, there is already a method for this in the repository